pdns/bpf-filter.cc

```diff
@@ -47,6 +47,51 @@ int bpf_create_map(enum bpf_map_type map_type, int key_size, int value_size,
   return syscall(SYS_bpf, BPF_MAP_CREATE, &attr, sizeof(attr));
 }
 
+int bpf_pin_map(int fd, const std::string& path)
```

Comment:
These `bpf_` helper functions can be `static`

Reply:
They can't because they have a declaration in `ext/libbpf/libbpf.h` :-/